geode-core/src/main/java/org/apache/geode/internal/cache/TXManagerImpl.java

```diff
@@ -669,8 +674,18 @@ private void closeListener(TransactionListener tl) {
     }
   }
 
-  private final ConcurrentMap<Thread, Boolean> transactionInternalSuspendedByThreads =
-      new ConcurrentHashMap<Thread, Boolean>();
+  private static final TXStateProxy pausedTXState = new PausedTXStateProxyImpl();
```

Comment:
I think the name "PAUSED" instead of "pausedTXState" might make clearer that this is a constant.